tests/api_test.py

```diff
@@ -50,3 +50,18 @@ def test_proper_error_on_missing_api_declaration():
         'tests/sample_schemas/missing_api_declaration/missing.json'
         in str(exc)
     )
+
+
+def test_ignore_absolute_paths():
```

Comment:
Let's stick a comment to this PR either in this test or in the code it covers so our future selves have some clue why we have this behavior :)